### ISSUE TITLE
TestCustom.yml breaks slufld if one does not move it away or change the secret

### ISSUE THREAD

**Issue by AndresTanasijczuk** [damienfrancois/slufl2#18]
The slufl2 package comes with a `TestCustom.yml` playbook and a `secret.yml` vault file used by that playbook. If one doesn't know the password to decrypt `secret.yml` and if one doesn't move away `TestCustom.yml` from `/etc/slufld.conf.d/`, slufld will break when trying to run `TestCustom.yml`, because it can not decrypt the `secret.yml` file.
When I install slufl2, I cleanup the `/etc/slufld.conf.d/` directory in order to remove these files. But for some reason these files appeared again in my installation and slufld was broken for more than 2 months without me noticing it until a user complained today that his key was not updated. One possible solution would be to move away `TestCustom.yml` into, for example, a `/etc/slufld.conf.d/test` subdirectory.

**Comment by damienfrancois** [comment]
Thanks for bringing that up. What about doing like for the `slufld.conf` file, have the RMP setup a `/etc/slufld.conf.d.examples` and adding instructions in the README to rename that to `/etc/slufld.conf.d` for initial tests? Would that be ok with you?

**Comment by AndresTanasijczuk** [comment]
Yes, that would be ok with me. And don't you need to add a file with the example password to decrypt secret.yml to the RPM?
